R/reporter-progress.R

```diff
@@ -191,7 +194,6 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
       } else if (expectation_skip(result)) {
         self$n_skip <- self$n_skip + 1
         self$ctxt_n_skip <- self$ctxt_n_skip + 1
-        self$ctxt_issues$push(result)
```

Comment:
Why would you remove this?

Reply:
Unrelated, but yes, I would remove this.

The skips are shown in the summary, that's more than enough. I have 100-s of skips in some packages, I don't care about them unless I do -- and then I go find them.

Reply:
I that seems reasonable, but lets check with @jennybc first.

Reply:
How would you know what file the skips occurred in?

Reply:
I would search for the skip message.

Reply:
I think I would be happier (or at least just as happy) seeing the skips at the end. I don't quite follow what's proposed for following up on skips, though, if they are unexpected.